packages/sdk/src/index.ts

```diff
@@ -51,6 +51,7 @@ export type Ops = {
   discoveryPlatformEndpoint?: string;
   timeout?: number;
   provider?: string;
+  mevProtectionProvider?: string;
```

Comment:
Please add it to the `@params` above

Reply:
pleaes add `enableMev?: boolean` to the global paramters